src/methods/photos/index.ts

```diff
@@ -63,7 +63,10 @@ export const getRandom = {
       },
     }),
   ),
-  handleResponse: castResponse<Photo.Random>(),
+  handleResponse: castResponse<
```

Comment:
I'm guessing there isn't an easy way to model this so the types convey that information instead? 🤔

Reply:
Hmm yeah I thought about that. Options:

- Overloads (a pain to work with, especially here because we're using function composition instead of function declarations)
- Two separate functions, one with count, one without

I'm not sure it's worth it at this point, we could see what feedback we get from users. WDYT?